scripts/loadEmptyPrefsSets.js

```diff
@@ -0,0 +1,127 @@
+/*!
+Copyright 2018 Raising the Floor US
+
+Licensed under the New BSD license. You may not use this file except in
+compliance with this License.
+
+You may obtain a copy of the License at
+https://github.com/GPII/universal/blob/master/LICENSE.txt
+*/
+
+// This script generates unique pref sets and loads them into CouchDB.
+//
+// To create 20 empty prefs sets, the sctipt needs to be called like this:
+//   * COUCHDB_URL=http://localhost:8058/gpii NEW_USERS=20 node scripts/loadEmptyPrefsSets.js
+//
+// The options to this script must be passed as environment variables, and they are:
+//   * COUCHDB_URL: This neeeds to be in the form "http://host/dbname", but in case
+//   you need to provide the credentials to CouchDB, you need to use the form
+//   "http://user:pass@host:port/dbname".
+//   * NEW_USERS: The amount of new empty preferences sets you want to create
+// Optional parameters
+//   * OUTPUT_FILENAME: The file to save the list of users created. By default,
+//   it will generate a file called "generated-prefsSets-<TIMESTAMP>", just provide a
+//   different value if you want it to be a different file.
+//   * SHOW_PROGRESS: Whether you want to see the users as they are created. Default is
+//   0 (no progress), set it to 1 if you want it to be the other way round.
+//
+// See this JIRA for more information: https://issues.gpii.net/browse/GPII-3381
+//
+
+"use strict";
+
+var process = require("process"),
+    fs = require("fs"),
+    fluid = require("infusion"),
+    gpii = fluid.registerNamespace("gpii"),
+    uuid = uuid || require("node-uuid");
+
+require("./shared/prefsSetsDbUtils.js");
+require("gpii-pouchdb");
+
+fluid.defaults("gpii.uuidLoader", {
+    gradeNames: ["gpii.pouch"],
+    outputFileName: process.env.OUTPUT_FILE || "generated-prefSets-" + new Date().toISOString(),
```

Comment:
Shall a file extension "txt" or "csv" be added to the default output file name so that it can be auto opened at double click?

Reply:
Good point!